portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,26 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+function audioPlayer() {
+  console.log('Running audio player.');
+  currentSong = 0;
+  player = document.getElementById('audio-player');
+  songs = document.querySelectorAll('#song-list li a');
+  console.log('Adding Event Listener to song links...');
```

Comment:
This doesn't seem very useful unless you expect the addEventListener code to fail or not run for some reason.

Reply:
When I was making the function I was running into issues and so I had added it there in order to debug and forgot to take it out. I realized that the problem was that inside the index.html the script tag was at the top and so it was not working since the document hadn't loaded.